packages/react-next/src/components/Link/examples/Link.Basic.Example.tsx

```diff
@@ -1,14 +1,17 @@
 import * as React from 'react';
 import { Link } from '@fluentui/react-next/lib/Link';
+import { useFocusRects } from '@fluentui/react-next/lib/Utilities';
 
 export const LinkBasicExample: React.FunctionComponent = () => {
+  useFocusRects();
```

Comment:
this should be called in LinkBase

Reply:
Why? This is not an example for `LinkBase` but for `Link`, it's just a basic example of it.

Reply:
`useFocusRects` should be baked in functionality within the component. meaning when user using `Link`, they should get focus rect without explicitly do anything extra